package.json

```diff
@@ -164,7 +164,7 @@
   "dependencies": {
     "@codingame/monaco-jsonrpc": "^0.3.1",
     "@docsearch/react": "^3.0.0-alpha.37",
-    "@influxdata/clockface": "^4.6.0",
+    "@influxdata/clockface": "^4.7.3",
```

Comment:
Before this is merged in, we should also get [clockface #794](https://github.com/influxdata/clockface/pull/794) merged in, so that the current item always shows first in the search bar.